### PR TITLE
Use the folder from which the R session is started as the renv project folder

### PR DESCRIPTION
**Pull request by dklein-pik**
## Purpose of this PR

Fixing https://github.com/remindmodel/development_issues/issues/305

The `.Rprofile` always sets the renv project folder to the folder from which the R session is stared. Prerequisite: the folder needs to contain the `.Rprofile`.
Result: if started in a run folder, renv uses the renv-library from the run folder and not from the REMIND main folder.

```
cd /p/tmp/dklein/remind-renvtest/
Rscript -e "setwd('output/testOneRegi'); system('Rscript -e \"renv::project()\"')"
cd output/testOneRegi/
Rscript -e "renv::project()"
```
Both return `[1] "/p/tmp/dklein/remind-renvtest/output/testOneRegi"`

## Type of change

- [x] Bug fix 

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [ ] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [ ] I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) where it was needed
- [ ] I adjusted `forbiddenColumnNames` in [readCheckScenarioConfig.R](https://github.com/remindmodel/remind/blob/develop/scripts/start/readCheckScenarioConfig.R) in case the PR leads to deprecated switches
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`)
- [ ] The changelog `CHANGELOG.md` [has been updated correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog)

## Further information (optional):

* Test runs are here: /p/tmp/dklein/remind-renvtest/output/testOneRegi

### CODE REVIEW

File: .Rprofile [hunk 1]
```diff
@@ -13,6 +13,9 @@ if (file.exists("renv.lock") && file.exists("README.md") && !file.exists("renv/o
 # do not check if library and renv.lock are in sync, because normally renv.lock does not exist
 options(renv.config.synchronized.check = FALSE)
 
+# always set the renv project to the current direcotry (formerly done by renv/activate.R under version 0.16.0)
```
Comment (orichters) [inline .Rprofile:16]:
```suggestion
# always set the renv project to the current directory (formerly done by renv/activate.R under version 0.16.0) 
```

File: scripts/start/run.R [hunk 2]
```diff
@@ -259,7 +259,7 @@ run <- function() {
   # make sure the renv used for the run is also used for generating output
   if (!is.null(renv::project())) {
     if (normalizePath(renv::project()) != normalizePath(outputdir)) {
-      warning("loaded renv=", normalizePath(renv::project()), " and outputdir=", normalizePath(outputdir), " must be equal.")
```
Comment (orichters) [inline scripts/start/run.R:262]: I strongly oppose that. If the output generation runs with some renv from the main folder or whatever, so what. This is not ideal for reproducibility (therefore: warn), but most often not a real drama. See also https://github.com/magpiemodel/magpie/pull/688

Comment (dklein-pik) [reply scripts/start/run.R:262]: No problem. I thought we wanted it this way once the bug is fixed, but I'll change it back.

**Review by orichters — APPROVED** [review]
thanks!

**Review by pascal-sauer — APPROVED** [review]

**Pull request merged**